src/runtime_src/core/pcie/driver/linux/xocl/subdev/icap.c

```diff
@@ -2519,6 +2519,34 @@ static inline int icap_xmc_free(struct icap *icap)
 	return err == -ENODEV ? 0 : err;
 }
 
+static bool check_group_topo_and_connectivity(struct icap *icap,
+	struct axlf *xclbin)
+{
+    struct connectivity *group_conn = icap->connectivity;
+	struct mem_topology *group_topo = icap->mem_topo;
```

Comment:
Please avoid this random mix of spaces and tabulation characters. :-(
Use a tool like `clang-format` or configure your editor or IDE.

Reply:
Will take care of that.